x-pack/plugin/security/src/main/java/module-info.java

```diff
@@ -49,6 +48,7 @@
     requires oauth2.oidc.sdk;
     requires org.slf4j;
     requires unboundid.ldapsdk;
+    // requires com.nimbusds.jose.jwt;
```

Comment:
If I leave this commented/omitted, IntelliJ is unhappy as classes from this module are used - makes sense, though Gradle doesn't find a problem(tests still fail on security manager stuff but that's separate). 

If I uncomment this, the Gradle build fails and IntelliJ is differently unhappy, because it can't find the module (despite happily taking me to the definition if I cmd-click it).